Webapp/sources/services/reaction.py

```diff
@@ -154,6 +178,16 @@ def list_active_in_workbook(
     return reaction_list
 
 
+def from_export_request(reaction_request) -> models.Reaction:
```

Comment:
Don't forget the docstring here

Reply:
Found this function wasn't actually used anywhere so removed it